lib/src/components/NxIconDropdown/NxIconDropdown.scss

```diff
@@ -21,7 +21,6 @@
   margin-left: 0;
 
   &.open {
-    background-color: var(--nx-swatch-grey-95);
-    border-color: var(--nx-swatch-grey-05);
+    background-color: var(--nx-swatch-blue-90);
```

Comment:
Also a variable already defined for it: `--nx-color-interactive-background-selected`

Reply:
> Also a variable already defined for it: `--nx-color-interactive-background-selected`

thanks for catching that! [cfef41f](https://github.com/sonatype/sonatype-react-shared-components/pull/990/commits/cfef41f493519717ac4aad50260c5ef22b4942fb)